tests/client/test_client.py

```diff
@@ -241,6 +242,29 @@ def test_delete_dataset(monkeypatch):
         rubrix.load(name=dataset_name)
 
 
+def test_dataset_copy(monkeypatch):
```

Comment:
What happens if the name of the copy already exists? Is it worth testing?

Reply:
An exception should be launched as default behaviour. I will test the case.

In future method versions, we can include  boolean `force_recreate`, or `overrides`, to configure default behaviours in those cases.